kornia/geometry/transform/affwarp.py

```diff
@@ -316,6 +328,8 @@ def scale(tensor: torch.Tensor, scale_factor: torch.Tensor,
     if center is None:
         center = _compute_tensor_center(tensor)
 
+    align_corners = _parse_align_corners(align_corners, Resample.get("bilinear"), old_default=False)
```

Comment:
Sorry. I commented on the wrong line number. XD

I mean I saw several places were hardcoded to "bilinear". Should we expose it as a function param?

Reply:
> I mean I saw several places were hardcoded to "bilinear"

In every such case, later `affine` without passing `mode` is used, whcih defaults to `"bilinear"`. Since the default for `align_corners` is dependent on the interpolation mode, I've used it as a hardcoded fallback here.

> Should we expose it as a function param?

Yes, see #811. This is out of scope for this PR.